content/manual/manual_circuit.rst

```diff
@@ -4,29 +4,30 @@ Circuit Construction
+Quantum circuits can be examined through two lenses: Positive Operator-Valued Measures (POVMs), encompassing quantum and classical information, and quantum operations, focusing on unitary transformations. POVMs offer insights into quantum states and measurement outcomes, but unitary operations are key for manipulating quantum states. A significant concept here is the global phase, which is crucial for understanding pure states and is observable when the system is coherently controlled. However, it is indistinguishable at the POVM level. For example, an Rz gate and a U1 gate give equivalent effects on the quantum state but have a different global phase, meaning their unitaries *look* different, and a controlled-Rz is different from a controlled-U1. A :py:class:`Circuit` will track global phase to make working with pure quantum processes easier. However, this becomes meaningless once measurements and other classical interactions are applied and have no impact on the instructions sent to a quantum device when we eventually run it.
 
 .. There is no strict notion of control-flow or branching computation within a :py:class:`Circuit`, meaning there is no facility to consider looping or arbitrary computation trees. This is likely to be an engineering limitation of all quantum devices produced in the near future, but this does not sacrifice the ability to do meaningful and interesting computation.
 
 .. Resource linearity - no intermediate allocation/disposal of (qu)bits
 .. Constructors (for integer-indexing)
 
-Given the small scale and lack of dynamic quantum memories for both devices and simulations, we assume each qubit and bit is statically registered and hence each :py:class:`Circuit` has the same number of inputs as outputs. The set of data units (qubits and bits) used by the :py:class:`Circuit` is hence going to be constant, so we can define it up-front when we construct one. We can also optionally give it a name for easy identification.
+Given the constraints of today's quantum computing, characterized by a limited number of qubits and static memory allocation, `pytket`'s :py:class:`Circuit` class is designed to work within these bounds by predefining the number of qubits and bits used. This setup means the number of inputs and outputs in a circuit is predefined and unchanging. Upon creation, each circuit's configuration is specified, and circuits can also be named for easier management in larger projects.  This process is illustrated in the following examples of initializing circuits in `pytket`:
```

Comment:
Perhaps "given the constraints of todays quantum devices"?

Reply:
done